packages/core/src/components/helpcenter-panel/helpcenter-panel.tsx

```diff
@@ -54,22 +54,24 @@ export class HelpcenterPanel {
           type="button"
           onClick={this.openClick}
           class={`open-button ${this.isOpen}`}
-          aria-expanded="false"
+          aria-expanded={this.isOpen === 'open' ? 'true' : 'false'}
           aria-haspopup="dialog"
+          aria-controls="dso-panel-wrapper"
         >
           <dso-icon icon="help"></dso-icon>
           <span>{this.label}</span>
         </button>
-        <div class={`wrapper ${this.visibility}`}>
+        <div id="dso-panel-wrapper" class={`wrapper ${this.visibility}`}>
           <div class="dimscreen" onClick={this.closeClick} />
           <div class={`iframe-container ${this.slideState}`} aria-live="polite">
-            {this.loadIframe ? <iframe src={this.url} /> : <div />}
+            {this.loadIframe ? <iframe src={this.slideState === 'open' ? this.url : undefined} /> : <div />}
```

Comment:
Gevolg van deze change is dat de `iframe` bij in en uitklappen wordt ingeladen, ook al is de `url` hetzelfde gebleven

Ik denk dat we zoeken naar een situatie waarin de een iFrame met een url 1 keer wordt ingeladen. En dat bij het wijzigen van de `@Prop() url` het iframe alleen wordt geladen als het paneel open is.

Reply:
`@Watch` op `url` en `isOpen` zodat een private variabele alleen veranderd wanneer het paneel open is en de url is veranderd.

Reply:
netjes 😄